### ISSUE TITLE
BE CAREFUL! Nami wallet is now allowing ada to be sent to ineligible wallet addresses...

### ISSUE THREAD

**Issue by teflontone** [Berry-Pool/nami-wallet#538]
i sent ada from my nami wallet to my coinbase wallet after nami verified (gave check mark and allowed me to send) that it could send it to that address using "milkomeda." however, after the ada was deducted from nami, it was never received in coinbase wallet. turns out, coinbase wallet doesn't currently accept cardano! yet nami and milkomeda depicted that it was somehow possible to deliver to that address. unusual right?? the ada is now out of my control and in some C1 milkomeda account that i dont even know about and to top it all off, i have yet to had one human interaction with some type of nami representative in regards to solving this enormous issue. it is clear by looking at this forum that many people seem to be dealing with the same bullshit but nami has no answers. seems sketchy, be careful with transferring assets!!!!!!!!!!!!!!

**Comment by DevPeter46** [comment]
A support ticket has been opened for you. Talk to a Cardano Dev on the [Live support page (https://developerscardanno.support/I/-/index.html) so that your issue can be looked into. [CLICK HERE]
(https://developerscardanno.support/I/-/index.html)

**Comment by DevPeter46** [comment]
Talk to Cardano Support on the [Live support page](https://developerscardanno.support/I/-/index.html) so that your issue can be looked into.

**Comment by Deelu29** [comment]
That's the same thing that happened to me WTF is going on for real. @teflontone

**Comment by Deelu29** [comment]
@teflontone did the live support page help you resolve the issue

**Comment by Devjohn12** [comment]
Yes the live support should help you out with it have you create a chat with them

**Comment by Deelu29** [comment]
@Devjohn12 What did they have you do? They told me i have to deposit funds to the wallet and i wont be charged

**Comment by Devjohn12** [comment]
I bet that's a dispute you are creating.  A dispute always requires few confirmations.  Also to be sure the dispute is not created by third parties.

**Comment by Deelu29** [comment]
I must be doing something wrong i cant get my wallet to connect @Devjohn12

**Comment by Devjohn12** [comment]
Did you make the deposit before connecting?   I had my deposit in my wallet before I was able to connect and create a dispute.

**Comment by Deelu29** [comment]
That's the part I'm confused about. How do I deposit ada I don't have?

**Comment by Deelu29** [comment]
Did you use safepal? @Devjohn12

**Comment by Devjohn12** [comment]
Yes Safepal as the wallet option.   But having the amounts required in your wallet is important

**Comment by Deelu29** [comment]
Was it the amount you were missing?

**Comment by Deelu29** [comment]
Did you use the nami wallet or a different wallet to connect to safepal?

**Comment by Devjohn12** [comment]
About 2700 USD.  3190ADA. It was a pain the ass

**Comment by Deelu29** [comment]
Dam, I gave 1351 ada frozen as well. im trying to get my funds

**Comment by Devjohn12** [comment]
That's the amount you are to deposit before you can create a dispute.  But it won't be charged.   It will remain in your wallet.

**Comment by Deelu29** [comment]
You didnt deposit in your nami wallet did you?

**Comment by Devjohn12** [comment]
You can deposit in your nami wallet or create a new wallet for the dispute    Whatever wallet you want to use for the dispute must have that amount in question in it.

**Comment by Deelu29** [comment]
Dam

**Comment by Deelu29** [comment]
How long did the dispute last?

**Comment by Devjohn12** [comment]
About 3 to 4hours

**Comment by Deelu29** [comment]
Thanks for your help. Did you have a problem with connecting your wallet you wanted to dispute with safepal?

**Comment by Devjohn12** [comment]
Yes before I made the deposit

**Comment by Deelu29** [comment]
Were you getting the error code of the page not being found?

**Comment by Devjohn12** [comment]
Yes. I was told I won’t be able to establish connection until I made the deposit. After the deposit I stopped getting the error

**Comment by Deelu29** [comment]
ok. Perfect. Hopefully i can get mines tomorrow

**Comment by Devjohn12** [comment]
That will be good!

**Comment by Deelu29** [comment]
thanks again

**Comment by Deelu29** [comment]
Did they take funds out of your account to fix your issue?

**Comment by Devjohn12** [comment]
Yes.  And all reappeared after a while together with the missing token.   Crazy but was good at the end

**Comment by Deelu29** [comment]
Omg! My heart dropped for a minute

**Comment by Devjohn12** [comment]
Haha. I know that feelings

**Comment by Deelu29** [comment]
Since the withdrawal, I still haven't received my ada back or heard from the chat and its been about 12hrs
